### DISCUSSION TITLE
Administrator Commands

### DISCUSSION THREAD

**Discussion by NoSiroSan** [n3d1117/chatgpt-telegram-bot#182] [Ideas]
Hello! You created a really very good bot, thank you for that!
But while the function of the administrator is not really functional. How about the following ideas for admin mode:

1./broadcast - a command that allows you to send a text message to everyone who uses the bot. For example "/broadcast bot will be turned off for a couple of hours to update!" etc

2./ban <user_id> - a command that prohibits the user from using the bot.

**Comment by ehsrrr** [comment]
I really need this

**Comment by stanislavlysenko0912** [comment]
It seems to me that you want to turn a bot in which people are trying to add a lot of useful features into a machine for making money from advertising? 🤔 
Or why are these features needed in an opensource project where you can add many other useful things

**Reply by NoSiroSan** [reply]
Hello! In fact, I wrote a broadcast function for the bot and I use it to report updates or problems on the server or other problems or innovations. But the command does not work as well as I planned(
And I don’t know why to ban users, it’s just a fun thing)
